stdlib/LinearAlgebra/test/special.jl

```diff
@@ -2,7 +2,7 @@
 
 module TestSpecial
 
-using Test, LinearAlgebra, Random
+using Test, LinearAlgebra, Random, StaticArrays
```

Comment:
I am not sure if we can add `StaticArrays` package using Pkg. So I will let others to weigh in on this. But if you are interested it might worth checking that change.